111.md

```diff
@@ -0,0 +1,183 @@
+NIP-111
+=======
+
+Addition Relay Information for NIP-11
+-------------------------------------
+
+`draft` `optional` `author:doc-hex`
+
+Relays may provide addition metadata to clients to inform them of
+more detailed limitations and policy choices. This is made available
+as additional fields in the a JSON document already defined by
+[NIP-11](11.md). All are optional and may be omitted.
+
+All numbers and values show in theses examples, are merely _examples_
+and do not constitute a default value or expectation. The purpose
+of this NIP is to allow clients to directly discover the actual
+values chosen by the relay operators.
+
+Background
+----------
+
+Support for this NIP should be documented in the `supported_nips` list
+of the same response.
+
+```json
+{
+...
+  supported_nips: <a list of NIP numbers supported by the relay, including this one>,
+...
+}
+```
+
+Server Limitations
+------------------
+
+These are limitations imposed by the relay on clients. Your client
+should expect that requests which exceed these *practical* limitations
+are rejected or fail immediately.
+
+```json
+{
+...
+  limitation: {
+        max_raw_json: 16384,
+        max_subscriptions: 20,
+        max_filters: 100,
+        max_limit: 5000,
+        subid_length: 100,
+        min_prefix: 4,
+  }
+...
+}
+```
+
+- `max_raw_json`: this is the maximum number of bytes for incoming JSON that the relay
+will attempt to decode and act upon. When you send large subscriptions, you will be
+limited by this value. It also effectively limits the maximum size of any event. Value is
```

Comment:
```suggestion
will attempt to decode and act upon. When you send large subscriptions or messages,
you will be limited by this value. It also effectively limits the maximum size of any event. It is
```

Reply:
kept orig because that's where I see the issue as a relay operator

Reply:
As a relay operator there's also issues with big events with lots of content or tags. You should reconsider. Check out https://nostr.io/stats for average event sizes.

Reply:
yeah, I'm adding those 2

Reply:
see 6b3038f..27ba498